client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -1,122 +1,162 @@
-                  </Grid>
-                  <Grid item container justify="center" className={classes.gridItemLocation}>
-                    <LocationOnIcon />
-                    <Typography>{sitter.location}</Typography>
-                  </Grid>
+      {sitter ? (
+        <Grid container className={classes.rootContainer} justify="space-evenly">
+          <Grid item xs={12} sm={12} md={7} lg={7}>
+            <Paper elevation={15}>
+              <Grid container direction="column">
+                <Grid item container className={classes.gridContainer}>
+                  <img
+                    src={baseUrl + sitter.backgroundPhotoName}
+                    alt="profile background"
+                    className={classes.backgroundImage}
+                  />
                 </Grid>
-                <Grid item container direction="column" className={classes.aboutMeContainer}>
-                  <Grid item>
-                    <Typography variant="h4" className={classes.profileTypography}>
-                      About me
-                    </Typography>
-                  </Grid>
-                  <Grid item>
-                    <Typography variant="body1">{sitter.aboutMe}</Typography>
+                <Grid item className={classes.gridItem}>
+                  <Grid item container direction="column" alignItems="center">
+                    <Grid item>
+                      <Avatar className={classes.avatar} alt="profile image" src={baseUrl + sitter.profilePhotoName} />
+                    </Grid>
+                    <Grid item>
+                      <Typography variant="h4" className={classes.profileTypography}>
+                        {sitter.firstName + ' ' + sitter.lastName}
+                      </Typography>
+                    </Grid>
+                    <Grid item>
+                      <Typography variant="body2">{sitter.description}</Typography>
+                    </Grid>
+                    <Grid item container justify="center" className={classes.gridItemLocation}>
+                      <LocationOnIcon />
+                      <Typography>{sitter.location}</Typography>
+                    </Grid>
                   </Grid>
-                  <Grid item>
-                    {sitter.aboutMeImgs.map((image) => (
-                      <img key={image} src={baseUrl + image} alt={image} className={classes.image} />
-                    ))}
+                  <Grid item container direction="column" className={classes.aboutMeContainer}>
+                    <Grid item>
+                      <Typography variant="h4" className={classes.profileTypography}>
+                        About me
+                      </Typography>
+                    </Grid>
+                    <Grid item>
+                      <Typography variant="body1">{sitter.aboutMe}</Typography>
+                    </Grid>
+                    <Grid item>
+                      {sitter.gallery
```

Comment:
```suggestion
                      {sitter.?gallery.map(....
```
You can directly map over this now 👍🏽